Release.toml

```diff
@@ -1,4 +1,4 @@
-version = "1.21.0"
+version = "1.21.1"
```

Comment:
Typically we would not add new settings to the API in a point release. These changes should target 1.22. But don't do the release version bump in a feature PR, it's not really related to your feature and creates some churn.